uber/models.py

```diff
@@ -296,6 +296,9 @@ def _ints(self, name, val):
 
     @suffix_property
     def _label(self, name, val):
+        if not val:
+            return ''
```

Comment:
Related question: do we WANT to do imports in those circumstances?  I guess allowing it is okay on the basis that this kind of import/export will be highly restricted to only People Who Know What They're Doing.  But in general, do we want to import an Attendee whose ``paid`` status is an unknown integer?  Do we want to import a Band whose panel status is gibberish?

The correct answer is probably "no, but let's leave it up to the admin", so leaving in this check is probably fine.  However, I guess maybe a future ticket would be a set of warnings that could be generated about all of the rows that were imported with unknown values.